use_case_examples/cifar_brevitas_training/evaluate_torch_cml.py

```diff
@@ -8,108 +8,82 @@
 
+        # Concrete ML inference only handles Numpy inputs
         numpy_input = input.detach().cpu().numpy()
 
-        # Compute Concrete ML output
-        y_cml_simulated = cml_inference(cml_model, numpy_input)
+        # Compute Concrete ML output using simulation
+        concrete_output_simulated = cml_model.forward(numpy_input, fhe="disable")
 
-        # y_cml_simulated to torch to device
-        y_cml_simulated = torch.tensor(y_cml_simulated).to(device)
+        concrete_output_simulated = torch.tensor(concrete_output_simulated).to(device)
 
-        # Compute torch loss
-        pred = output.data.argmax(1, keepdim=True)
-        correct = pred.eq(target.data.view_as(pred)).sum()
-        prec1 = 100.0 * correct.float() / input.size(0)
+        # Compute Torch top accuracies
+        torch_top_1, torch_top_5 = accuracy(torch_output, target, topk=(1, 5))
 
-        _, prec5 = accuracy(output, target, topk=(1, 5))
+        torch_top_1_batches.append(torch_top_1.item())
+        torch_top_5_batches.append(torch_top_5.item())
 
-        top1_torch.append(prec1.item())
-        top5_torch.append(prec5.item())
+        # Compute Concrete ML top accuracies
+        concrete_top_1, concrete_top_5 = accuracy(concrete_output_simulated, target, topk=(1, 5))
 
-        # Compute Concrete ML loss
-        pred = y_cml_simulated.data.argmax(1, keepdim=True)
-        correct = pred.eq(target.data.view_as(pred)).sum()
-        prec1 = 100.0 * correct.float() / input.size(0)
+        concrete_top_1_batches.append(concrete_top_1.item())
+        concrete_top_5_batches.append(concrete_top_5.item())
 
-        _, prec5 = accuracy(y_cml_simulated, target, topk=(1, 5))
+    print("Torch accuracy top1:", np.mean(torch_top_1_batches))
+    print("Concrete ML accuracy top1:", np.mean(concrete_top_1_batches))
 
-        top1_cml.append(prec1.item())
-        top5_cml.append(prec5.item())
+    print("Torch accuracy top5:", np.mean(torch_top_5_batches))
+    print("Concrete ML accuracy top5:", np.mean(concrete_top_5_batches))
 
-    print("Torch accuracy top1:", np.mean(top1_torch))
-    print("Concrete ML accuracy top1:", np.mean(top1_cml))
 
-    print("Torch accuracy top5:", np.mean(top5_torch))
-    print("Concrete ML accuracy top5:", np.mean(top5_cml))
+def main(args):
+    rounding_threshold_bits_list = args.rounding_threshold_bits
 
-
-def main(rounding_threshold_bits_list):
     model = cnv_2w2a(False)
-    # MPS option is supported by macOS with Apple Silicon or AMD GPUs
-    device = (
-        "cuda"
-        if torch.cuda.is_available()
-        else "mps"
-        if torch.backends.mps.is_available()
-        else "cpu"
-    )
+
+    # Add MPS (for macOS with Apple Silicon or AMD GPUs) support when error is fixed. For now, we
+    # observe a decrease in torch's top1 accuracy when using MPS devices
+    # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3953
+    device = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
Some issues were found with the use of MPS devices : torch's accuracy is decreased (https://github.com/zama-ai/concrete-ml-internal/issues/3953). For now, disabling the use of MPS device